core/src/main/scala/kafka/utils/ToolsUtils.scala

```diff
@@ -16,10 +16,36 @@
 */
 package kafka.utils
 
+import java.io.File
 import joptsimple.OptionParser
+import java.net.URI
+import java.security.URIParameter
+import javax.security.auth.login.Configuration
+
+import kafka.common.KafkaException
 
 object ToolsUtils {
 
+  def isSecure(loginConfigFile: String): Boolean = {
```

Comment:
The functionality isn't specific to ZK, although currently it is only used by ZK-specific code. I've left as is for now.